result-1

```diff
@@ -0,0 +1 @@
+/nix/store/x4kvk5qc73l2dpn0zjzqdmn47y75f1hg-default-Setup
```

Comment:
What is this?

Reply:
Ah, AFAIU it is a symlink that was committed by mistake. Correct?

Reply:
Yep, correct. We need to remove this and probably add a gitignore with `result*/`